common/app/services/newsletters/NewsletterSignupAgent.scala

```diff
@@ -86,6 +86,14 @@ class NewsletterSignupAgent(newsletterApi: NewsletterApi) extends GuLogging {
     }
   }
 
+  def getNewsletters(): Either[String, List[NewsletterResponse]] = {
+    newslettersAgent.get() match {
+      case Left(err) => Left(err)
+      case Right(newsletters) =>
+        Right(newsletters)
+    }
+  }
```

Comment:
Not really sure why we use this pattern in this class - i.e. pattern match and then return  the input unchanged. Any ideas?

Reply:
I don't know either! I will replace this with `newslettersAgent.get()` and same for above

Reply:
https://github.com/guardian/frontend/pull/24147/commits/64b511ea8aba0fec5c1f7bbe8b3f007ba844faa1